src/pages/who-we-are/_data.tsx

```diff
@@ -1,227 +1,212 @@
-import React, { ReactElement } from 'react'
-import { Localize } from 'components/localization'
 import Shield from 'images/svg/who-we-are/shield.svg'
 import Star from 'images/svg/who-we-are/star.svg'
 import People from 'images/svg/who-we-are/people.svg'
 import Hands from 'images/svg/who-we-are/hands.svg'
 import { all_offices, all_countries } from 'pages/careers/_model/_locations/_locations'
+import { TString } from 'types/generics'
+
+const number_of_offices = all_offices.length.toString()
```

Comment:
@mohsen-deriv mohsen could u pls check this too, is that ok to use it like this?

Reply:
yes it is, for now let's use this approach. thanks :pray:

Reply:
Thanks for checking :woman_dancing: